src/OpenLoco/src/Windows/ObjectSelectionWindow.cpp

```diff
@@ -239,6 +241,8 @@ namespace OpenLoco::Ui::Windows::ObjectSelectionWindow
         objectImage,
     };
 
+    static constexpr uint8_t kNumSecondaryTabs = 8;
```

Comment:
should this be the max number? ie `kMaxNumSecondaryTabs`

Reply:
Sure, that would be a better name.